CONTRIBUTING.md

```diff
@@ -62,6 +64,29 @@ npm link cf-buttons
 
 Now `~/Projects/owning-a-home/node_modules/cf-buttons` will be a symlink pointing to the `~/Projects/capital-framework/tmp/cf-buttons` directory. Whenever you rebuild (`npm run build`, see above) the CF components, your local owning-a-home project will reference your local `tmp/` version of cf-buttons.
 
+### Updating Documentation
+
+Documentation is maintained in each component's folder's `usage.md` file.
+Changes to documentation are made on the `canary` branch using the workflow above
+and pulled in the `gh-pages` branch using a manual process.
```

Comment:
It might be good to include a quick note about why we do this process. Something like "The `usage.md` files are then rendered as separated pages on [Capital Framework's website](https://cfpb.github.io/capital-framework/components/)".